### DISCUSSION TITLE
This user account is blocked

### DISCUSSION THREAD

**Discussion by MBT-Logistics** [vercel/community#4678] [Account Issue]
### Question

Dear Vercel Support Team,
I'm writing to request the unblocking of my Vercel account, which is currently displaying the error message: "This User Account is Blocked."
Case Number:- 00165372

I'm committed to resolving any issues promptly to ensure my account complies with Vercel's policies. Please review my account and provide instructions for resolution.
Thank you for your assistance.

**Comment by niruallen** [comment]
Same
case number is 00169143

**Comment by KierynMo** [comment]
Mine is blocked as well.
Case number 00167272. Its been over a week with no word.
